src/components/container/GlucoseChart/GlucoseChart.stories.tsx

```diff
@@ -44,28 +67,18 @@ export const Default = {
         showStats: true,
         averageGlucoseLineColor: undefined
     },
-    argTypes: {
-        colorScheme: {
-            name: 'color scheme',
-            control: 'radio',
-            options: ['auto', 'light', 'dark']
-        },
-        state: {
-            name: 'state',
-            control: 'radio',
-            options: ['loading', 'no data', 'with data', 'live']
-        },
-        withMeals: {
-            name: 'with meals'
-        },
-        showStats: {
-            name: 'show stats'
-        },
-        averageGlucoseLineColor: {
-            name: 'average glucose line color',
-            control: 'color'
-        }
-    },
+    argTypes: sharedArgTypes,
     render: render
 };
 
+export const Live = {
```

Comment:
What does having this additional export gain us?  The default export above already supports the "live" state.